addons/Candle-manager-addon.json

```diff
@@ -23,7 +23,11 @@
       "api": {
         "min": 2,
         "max": 2
+      },
+      "gateway": {
+        "min": "0.10.0",
+        "max": "*"
       }
     }
   ]
-}
+}
```

Comment:
nit: These are all showing as missing a newline at the end of the file.

Reply:
Fixed.